### PR TITLE
RoR - Validations and Specs

### PR DESCRIPTION
**Pull request by Rebelzob**
# Blog App 📓

- Added validations
  - For the User model:
     >Name must not be blank.
     >PostsCounter must be an integer greater than or equal to zero.

  - For the Post model:
     >Title must not be blank.
     >Title must not exceed 250 characters.
     >CommentsCounter must be an integer greater than or equal to zero.
     >LikesCounter must be an integer greater than or equal to zero.

- Added specs for Models and Validations

### CODE REVIEW

File: spec/models/comment_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  fixtures :users, :posts
+
+  describe 'associations' do
+    it { should belong_to(:user) }
+    it { should belong_to(:post) }
+  end
+
+  it 'update the comments_counter of the post' do
+    comment = Comment.create(user: users(:one), post: posts(:one), text: 'Hello')
+    expect(comment.post.comments_counter).to eq(6)
```
Comment (Davidosky007) [inline spec/models/comment_spec.rb]: * I get an error here when i try to run your tests , kindly fix it 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Rebelzob 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* Validations for all model written 👍 
* Unit test for all models written 👍 
* Good looking readme 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/comment_spec.rb (hunk 1 above)
Comment (Rebelzob) [reply spec/models/comment_spec.rb]: @Davidosky007 Thanks for the review!

- actually in my terminal, if I change the text I get a different output.
image

- and when I leave it as it was all tests pass in my terminal
image

**Review by OyePriscilla — APPROVED** [review]
Hi @Rebelzob 👏 👏 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
All written tests passed ✔️ 

Untitled

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**